forgerock-integration-tests/src/androidTest/java/org/forgerock/android/auth/callback/DeviceProfileCollectorCallbackAndroidTest.java

```diff
@@ -111,17 +122,37 @@ public void testLocation() throws Exception {
 
         assertTrue(content.contains("identifier"));
 
-        int backgroundLocationPermissionApproved =
-                ActivityCompat.checkSelfPermission(context,
-                        Manifest.permission.ACCESS_BACKGROUND_LOCATION);
+        boolean locationPermissionGranted = false;
+
 
-        if (!isEmulator() && backgroundLocationPermissionApproved >= 0) {
+        // If the location services are NOT enabled skip the rest of the test...
```

Comment:
Attempted to fix random failures on BitBar where for some reason on some divide location data is not available...